python/mxnet/contrib/svrg_optimization/README.md

```diff
@@ -0,0 +1,41 @@
+## SVRG Optimization in Python Module
+
+### Problem Description
+SVRG stands for Stochastic Variance Reduced Gradient, which was first introduced in the paper _Accelerating Stochastic 
```

Comment:
This file should be removed and SVRG description and characteristics should go into the documentation of the SVRG optimizer.